setup.py

```diff
@@ -39,24 +40,44 @@
 
 DOCLINES = __doc__.split('\n')
 
+TFA_NIGHTLY = 'tfa-nightly'
+TFA_RELEASE = 'tensorflow-addons'
+
+if '--nightly' in sys.argv:
+    project_name = TFA_NIGHTLY
+    nightly_idx = sys.argv.index('--nightly')
+    sys.argv.pop(nightly_idx)
+else:
+    project_name = TFA_RELEASE
+
+# Version
 version = {}
 base_dir = os.path.dirname(os.path.abspath(__file__))
 with open(os.path.join(base_dir, "tensorflow_addons", "version.py")) as fp:
     # yapf: disable
     exec(fp.read(), version)
     # yapf: enable
 
+if project_name == TFA_NIGHTLY:
+    version['__version__'] += datetime.strftime(datetime.today(), "%Y%m%d")
+
+# Dependencies
 REQUIRED_PACKAGES = [
     'six >= 1.10.0',
 ]
 
-if '--nightly' in sys.argv:
-    project_name = 'tfa-nightly'
-    nightly_idx = sys.argv.index('--nightly')
-    sys.argv.pop(nightly_idx)
-    version['__version__'] += datetime.strftime(datetime.today(), "%Y%m%d")
-else:
-    project_name = 'tensorflow-addons'
+if project_name == TFA_RELEASE:
+    # TODO: remove if-else condition when tf supports package consolidation.
+    if platform.system() == 'Linux':
+        REQUIRED_PACKAGES.append('tensorflow-gpu >= 2.0.0-rc0')
```

Comment:
Is the version requirement `>=` correct? For example if a user froze `tensorflow_addons` to a version built against TensorFlow 2.0 and then update to TensorFlow 2.1, will it work correctly?

Reply:
Even though backwards compatible API guarantees should be there, there may new features we're using so I think this is a good catch.

Should be `==`

Reply:
Thanks, done :-)